setup.py

```diff
@@ -71,7 +71,16 @@
             'django-s3-file-field[minio]',
             'ipython',
             'tox',
-        ]
+        ],
+        'test': [
+            'factory-boy',
+            'pytest',
+            'pytest-cov',
+            'pytest-django',
+            'pytest-factoryboy',
+            'pytest-mock',
+            'requests',
+        ],
```

Comment:
you can remove the list from tox.ini then and just add `test` to go along with `dev` as
```
diff --git a/tox.ini b/tox.ini
index b4e8fc6..8e6a363 100644
--- a/tox.ini
+++ b/tox.ini
@@ -48,14 +48,7 @@ passenv =
     DJANGO_DANDI_SCHEMA_VERSION
 extras =
     dev
-deps =
-    factory-boy
-    pytest
-    pytest-cov
-    pytest-django
-    pytest-factoryboy
-    pytest-mock
-    requests
+    test
 commands =
     pytest {posargs}
 
```